consensus/polybft/stake_manager.go

```diff
@@ -150,51 +150,6 @@ func (s *stakeManager) init(blockchain blockchainBackend, dbTx *bolt.Tx) error {
 		"last saved", validatorSet.BlockNumber,
```

Comment:
`init` function should be just:

```
func (s *stakeManager) init(blockchain blockchainBackend, dbTx *bolt.Tx) error {
	_, err := s.getOrInitValidatorSet(dbTx)

	return err
}
```

This is only used when chain is starting as new to insert the first validator set to db, so that it can be used by other functions in `stakeManager`. Updating it is the job of `EventProvider`.

Reply:
Fixed 5616959fee08353bcb09aa41a7054f4b73251e49